exchanges/gateio/gateio_websocket.go

```diff
@@ -52,20 +53,23 @@ const (
 
 	subscribeEvent   = "subscribe"
 	unsubscribeEvent = "unsubscribe"
+
+	spotOrderbookUpdateLimit = 100
 )
 
 var defaultSubscriptions = subscription.List{
 	{Enabled: true, Channel: subscription.TickerChannel, Asset: asset.Spot},
 	{Enabled: true, Channel: subscription.CandlesChannel, Asset: asset.Spot, Interval: kline.FiveMin},
-	{Enabled: true, Channel: subscription.OrderbookChannel, Asset: asset.Spot, Interval: kline.HundredMilliseconds},
+	// spot.order_book_update is locked to 100ms updates and 100 book size
+	{Enabled: true, Channel: subscription.OrderbookChannel, Asset: asset.Spot, Interval: kline.HundredMilliseconds, Levels: spotOrderbookUpdateLimit},
+	{Enabled: false, Channel: spotOrderbookTickerChannel, Asset: asset.Spot, Interval: kline.Interval(10 * time.Millisecond), Levels: 1},
```

Comment:
🐒 Feels weird to have a non-const Interval that isn't too strange to put in kline_types.
Can we have `kline.TenMilliseconds` ?